<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FailoverPriorityPickerWidget</class>
 <widget class="QWidget" name="FailoverPriorityPickerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>35</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pickerPage">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>16</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>16</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <item>
        <widget class="QLabel" name="setPriorityLabel">
         <property name="text">
          <string>Set priority:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="neverButton">
         <property name="text">
          <string>No Failover</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="lowButton">
         <property name="text">
          <string>Low</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mediumButton">
         <property name="text">
          <string>Medium</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="highButton">
         <property name="text">
          <string>High</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="placeholderPage">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>16</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>16</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <item>
        <widget class="QLabel" name="placeholderLabel">
         <property name="text">
          <string>Select some devices to setup failover priority</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>105</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>neverButton</tabstop>
  <tabstop>lowButton</tabstop>
  <tabstop>mediumButton</tabstop>
  <tabstop>highButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
